sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureActiveDirectoryConfiguration.java

```diff
@@ -172,4 +206,21 @@ protected void configure(HttpSecurity http) throws Exception {
                 .oidcUserService(oidcUserService);
         }
     }
+
+    /**
+     * Sample configuration to make RefreshTokenGrantRequestEntityConverter take effect.
+     * TODO: spring-security can't inject OAuth2AuthorizedClientManager to OAuth2AuthorizedClientArgumentResolver
+     *  Future versions may support this , this may be a better solution
+     *  issues : https://github.com/spring-projects/spring-security/issues/8700
```

Comment:
Use one space instead of two after `*`.
No space after `issues`.
It will be `issue` instead of `issues`.